ribbon/option.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#----------------------------------------------------------------------------
+# Created By: Steven (steven@ribbon.finance)
+# Created Date: 04/04/2022
+# version ='0.01'
+# ---------------------------------------------------------------------------
+''' Abstract class for contract factory '''
+# ---------------------------------------------------------------------------
+
+# ---------------------------------------------------------------------------
+# Imports
+# ---------------------------------------------------------------------------
+from contract import ContractFactory
+
+# ---------------------------------------------------------------------------
+# Constants
+# ---------------------------------------------------------------------------
+DEFAULT_ABI_LOCATION = 'abis/oToken.json'
+
+# ---------------------------------------------------------------------------
+# oToken Contract Factory
+# ---------------------------------------------------------------------------
+class oTokenFactory(ContractFactory):
```

Comment:
Just like above, we could call this `oTokenConnection`.
Or if we call `ContractFactory` simply `Contract`, which is also fine, this would be then `oTokenContract`.

Reply:
Changed to `oTokenContract`